docs/tools/dotnet-embedding/limitations.md

```diff
@@ -44,4 +44,8 @@ Since we cannot generate accurate nullability annotations in the header files an
 
 Currently .NET Embedding does not support bitcode on iOS, which is enabled for some Xcode project templates. This will have to be disabled to successfully link generated frameworks.
 
+For iOS bitcode is optional to submit Apps to Apple's AppStore. Xamarin.iOS does not support it for iOS (only) since the generated bitcode is “inline assembly”. This provides no benefit on the iOS platform, i.e. it can’t be optimized (server-side), but makes binaries larger and build times longer.
```

Comment:
should "Apps" be uppercase? I think "apps".

> (only)
seems confusing. I would just remove it or clarify it further 

For all these, should they perhaps be bullets? When previewed it's all one big paragraph, and I liked them better separately. Easier to scan (which is how docs are typically read, right?).

The initial dependent clauses need a comma on each, like... "For iOS, bitcode" and "For tvOS and watchOS, bitcode"
server-side), but makes binaries larger and build times longer.

Reply:
Don't trust anything I write in English, or in French for that matter ;-)
Even less for formatting... I would have more success using a RNG to place them.

I remember some conversation about "Apps" with Vincent and I think it's not very consistent across Apple. Funnily that's something easier in French - which is quite uncommon :)